package-lock.json

```diff
@@ -7706,10 +7722,9 @@
       "license": "MIT"
     },
     "node_modules/@types/react": {
-      "version": "18.2.11",
-      "resolved": "https://registry.npmjs.org/@types/react/-/react-18.2.11.tgz",
-      "integrity": "sha512-+hsJr9hmwyDecSMQAmX7drgbDpyE+EgSF6t7+5QEBAn1tQK7kl1vWZ4iRf6SjQ8lk7dyEULxUmZOIpN0W5baZA==",
-      "license": "MIT",
```

Comment:
I am just curious about the license removal. What is the new license type?

Reply:
> I am just curious about the license removal. What is the new license type?

I didn't notice this. Probably this is an automatic update when I install or uninstall some packaage